From 640736d1f20fa36e2cf5ef02450dc5caa7e4893d Mon Sep 17 00:00:00 2001
From: Billy Tsai <billy_tsai@aspeedtech.com>
Date: Mon, 15 Jun 2026 18:25:27 +0800
Subject: [PATCH] cmd: aspeed: ltpi: Fix LVDS_TX_DS_EN bit and drop bogus
 2LTPI_EN macro

The LTPI_SW_FORCE_VAL register has no 2LTPI force bit, so the
REG_LTPI_SW_FORCE_2LTPI_EN definition is bogus. Remove it and correct
REG_LTPI_SW_FORCE_LVDS_TX_DS_EN to BIT(1), which is the bit it actually
occupies.

Upstream-Status: Backport [SDK v11.03 migration]
Change-Id: I42af988ea1b5a990e8bf897e04e3a8e4fc08b101
Signed-off-by: Billy Tsai <billy_tsai@aspeedtech.com>
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 cmd/aspeed/ltpi/ltpi_top.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cmd/aspeed/ltpi/ltpi_top.h b/cmd/aspeed/ltpi/ltpi_top.h
index e5304e3240..93928ecdc6 100644
--- a/cmd/aspeed/ltpi/ltpi_top.h
+++ b/cmd/aspeed/ltpi/ltpi_top.h
@@ -93,7 +93,6 @@
 #define   REG_LTPI_STRAP_1700_EN		BIT(0)
 #define LTPI_SW_FORCE_EN			0x010
 #define LTPI_SW_FORCE_VAL			0x014
-#define   REG_LTPI_SW_FORCE_LVDS_TX_DS_EN	BIT(2)
-#define   REG_LTPI_SW_FORCE_2LTPI_EN		BIT(1)
+#define   REG_LTPI_SW_FORCE_LVDS_TX_DS_EN	BIT(1)
 #define   REG_LTPI_SW_FORCE_1700_EN		BIT(0)
 #endif	/* _LPTI_TOP_H_ */
-- 
2.43.0

